var/spack/repos/builtin/packages/keepalived/package.py

```diff
@@ -12,7 +12,7 @@ class Keepalived(AutotoolsPackage):
     maintain and manage loadbalanced server pool according their health
     """
 
-    homepage = "http://www.keepalived.org"
+    homepage = "https://www.keepalived.org"
```

Comment:
Did you check the `http` below? The `homepage` is information for users, but we actually use `url` to download tarballs - so if possible it'd be better to update that too.

Reply:
I did check quite a few of these (maybe not this one in particular) and found that for many http only worked. Since I did this all manually and it took hours to check and copy paste URLs I decided to scope my fixes to just homepages since that’s what repology cares about. I did have to manually open about 400 files and check and copy paste, yes, lol.

Reply:
I did check quite a few of these (maybe not this one in particular) and found that for many http only worked. Since I did this all manually and it took hours to check and copy paste URLs I decided to scope my fixes to just homepages since that’s what repology cares about. I did have to manually open about 400 files and check and copy paste, yes, lol.

Reply:
Ok, fair enough. This is anyhow an improvement over current state so LGTM.